src/graphics/internal_canvas.rs

```diff
@@ -398,7 +405,7 @@ impl<'a> InternalCanvas<'a> {
         let uniforms = InstanceUniforms {
             transform: (self.transform
                 * glam::Mat4::from(
-                    DrawUniforms::from_param(&param.src(Rect::one()), [1., 1.].into()).transform,
+                    DrawUniforms::from_param(&param.src(Rect::one()), None).transform,
```

Comment:
Please add a comment like "image_scale is None here so that src isn't used for scaling the geometry" for later generations to better understand why there is a scale down there but not simply up here.